README.md

```diff
@@ -23,73 +70,5 @@ REACT_APP_MAPBOX_STYLE_URL=
 REACT_APP_MAPBOX_API_KEY=
 
 NODE_ENV=
```

Comment:
you can probably template this to NODE_ENV=development

Reply:
templated to NODE_ENV=development